toolinfo.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name" : "interaction-timeline",
+    "title" : "Interaction Timeline",
+    "description" : "Shows a chronologic history for two users on pages where they have both made edits.",
+    "url" : "https://meta.wikimedia.org/wiki/Community_health_initiative/Interaction_Timeline",
+    "keywords" : "anti-harassment, javascript, php, wikipedia",
+    "author" : "David Barratt",
```

Comment:
Can we have multiple authors here? If so, we should probably add "Dayllan Maza" or find a better way to indicate multiple people's contributions.